content/content-tool/tool/src/webapp/vm/content/sakai_resources_zipFinish.vm

```diff
@@ -0,0 +1,59 @@
+<!-- chef_resources_zipFinish.vm, version: $Revision:  $, use with org.sakaiproject.tool.content.ResourcesAction.java -->
+<div class="portletBody specialLink">
+	#if ($alertMessage)<div class="alertMessage">$tlang.getString("label.alert") $validator.escapeHtml($alertMessage)</div><div style="display:block;clear:both" ></div>#end
+	<div class="alertMessage">$tlang.getString("zip.sure")</div><div style="display:block;clear:both" ></div>
+	<h3>
+        $tlang.getString("zip.zip")
+	</h3>
+	<form action="#toolForm("ResourcesAction")" name="deleteFileForm" method="post" >
+		<input type="hidden" name="collectionId" value="$validator.escapeHtml($collectionId)" />
+		#if ($zipItems.size()>0)
```

Comment:
Have a `#else# clause with error, or why bother doing the test as the user will just see an empty page with no cancel button?